### PR TITLE
Merge xcode11 into master.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: Make.versions [hunk 1]
```diff
@@ -43,5 +43,5 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=12.15.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=5.15.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=13.4.0.$(IOS_COMMIT_DISTANCE)
+MAC_PACKAGE_VERSION=6.4.0.$(MAC_COMMIT_DISTANCE)
```
Comment (spouliot) [inline Make.versions]: `master` should be `x.3.0` (odd) until we branch `d16-4` so it's easy to distinguish the branch from the number

Considering that `d16-4` release date is likely after one (or maybe two) Xcode 11.x release we might even want to keep more _digits_ available. I suggest

* `d16-3` -> x.2.0
* `xcode11.1` -> x.4.0 (should be branched from `d16-3` after the `xcode11` merge)
* `xcode11.2` -> x.6.0
* `d16-4` -> x.8.0
* `master` (post 16-4 branch) -> x.9.0

c.c. @chamons

File: Make.config [hunk 2]
```diff
@@ -40,9 +43,9 @@ IOS_PACKAGE_VERSION_BUILD=$(IOS_COMMIT_DISTANCE)
...
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=10.2
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_10.2.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode102.app/Contents/Developer
+XCODE_VERSION=11.0
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11_GM_Seed.xip
+XCODE_DEVELOPER_ROOT=/Applications/Xcode11-GM.app/Contents/Developer
 
 XCODE94_VERSION=9.4
 XCODE94_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_9.4.xip
```
Comment (spouliot) [inline Make.config]: is there a point in keeping 9.4 around ?
can be fixed in a subsequent PR

Comment (rolfbjarne) [reply Make.config]: It's on my todo list to remove the Xcode 9.4 dependency, I just didn't want to complicate this PR even more.

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-7003/1/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-7003/1/display/redirect) (on internal Jenkins) failed in stage(s) 'Running XM tests on '10.15', Running XM tests on '10.15'' 🔥

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-7003/1//console)
✅ Packages: 
* [xamarin.ios-13.4.0.16.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/package/xamarin.ios-13.4.0.16.pkg)
* [xamarin.mac-6.4.0.16.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/package/xamarin.mac-6.4.0.16.pkg)
* [xamarin.ios-13.4.0.16.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/package/notarized/xamarin.ios-13.4.0.16.pkg)
* [xamarin.mac-6.4.0.16.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/package/notarized/xamarin.mac-6.4.0.16.pkg)

✅ [API Diff (from stable)](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/jenkins-results/api-diff/index.html)
✅ [API Diff (from PR only)](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/jenkins-results/apicomparison/api-diff.html) (no change)
✅ [Generator Diff](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/jenkins-results/generator-diff/index.html) (no change)
🔥 [Xamarin.Mac tests on 10.15 failed: Xamarin.Mac tests on macOS 10.15 failed (introspection)](https://jenkins.internalx.com/job/macios/job/PR-7003/1/display/redirect) 🔥
🔥 [Test run failed](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/0186f0ec1af2234a2cbe4945ddd4efa4fc0a6059/1/jenkins-results/tests/index.html) 🔥

# Test results

386 tests' device not found, 163 tests passed.

File: Make.config (hunk 2 above)
Comment (mandel-macaque) [reply Make.config]: @spouliot AFAIK I added a check but just for Catalina.

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-7003/2/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-7003/2/display/redirect) (on internal Jenkins) failed in stage(s) 'Build' 🔥 : hudson.AbortException: script returned exit code 2

🔥 [Build failed](https://jenkins.internalx.com/job/macios/job/PR-7003/2//console) 🔥

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-7003/4/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-7003/4/display/redirect) (on internal Jenkins) failed in stage(s) 'Build' 🔥 : hudson.AbortException: script returned exit code 2

🔥 [Build failed](https://jenkins.internalx.com/job/macios/job/PR-7003/4//console) 🔥

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-7003/5/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-7003/5/display/redirect) (on internal Jenkins) failed in stage(s) 'Test run, Test run' 🔥

✅ [Build succeeded](https://jenkins.internalx.com/job/macios/job/PR-7003/5//console)
✅ Packages: 
* [xamarin.ios-13.3.0.5.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/package/xamarin.ios-13.3.0.5.pkg)
* [xamarin.mac-6.3.0.5.pkg (Not notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/package/xamarin.mac-6.3.0.5.pkg)
* [xamarin.ios-13.3.0.5.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/package/notarized/xamarin.ios-13.3.0.5.pkg)
* [xamarin.mac-6.3.0.5.pkg (Notarized)](https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/package/notarized/xamarin.mac-6.3.0.5.pkg)

✅ [API Diff (from stable)](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/jenkins-results/api-diff/index.html)
⚠️ [API Diff (from PR only)](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/jenkins-results/apicomparison/api-diff.html) (🔥 breaking changes 🔥)
ℹ️ [Generator Diff](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/jenkins-results/generator-diff/index.html) (please review changes)
🔥 [Test run failed](http://xamarin-storage/jenkins/xamarin-macios/PR-7003/e7a5d56dd23c46e08729ae9c8eaafc7844285ba1/5/jenkins-results/tests/index.html) 🔥

# Test results

<details>
<summary>1 tests failed, 386 tests' device not found, 162 tests passed.</summary>

## Failed tests

 * monotouch-test/watchOS 32-bits - simulator/Debug: Crashed
</details>

**Pull request merged**